pkg/pgp/pgp_test.go

```diff
@@ -96,7 +96,7 @@ func TestTestEncryptNoEncryptionKey(t *testing.T) {
 	assert.Nil(t, err)
 	err = TestEncrypt(entity)
 	assert.NotNil(t, err)
-	assert.Regexp(t, `cannot encrypt a message .+ no encryption keys`, err)
+	assert.Regexp(t, `error setting up encryption for PGP message: openpgp: invalid argument: cannot encrypt a message to key id .+ because it has no valid encryption keys`, err)
```

Comment:
Nice update! From (version) 2021 to 2023!

Reply:
Should be fine, since we don't rely on terraform encryption, right?

Reply:
Yes, it should be no issue.